### ISSUE TITLE
Uncaught TypeError: Cannot read property 'absolutePath' of null

### ISSUE THREAD

**Issue by onlyskyguy** [mulesoft/api-workbench#78]
1. Following tutorial at [http://apiworkbench.com/docs]{http://apiworkbench.com/docs}
2. At the section "Filling method bodies and responses", for step "To add a response body, make sure the focus is on 200: and choose Create new Response Body out of the available actions.", when I click on "Create new Response Body" I get this error message.

**Atom Version**: 1.3.2
**System**: Ubuntu 15.10
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.9

### Stack Trace

Uncaught TypeError: Cannot read property 'absolutePath' of null

```
At /home/orestis/.atom/packages/api-workbench/main.js:3

TypeError: Cannot read property 'absolutePath' of null
    at t.typeId (/home/orestis/.atom/packages/api-workbench/main.js:3:9393)
    at t.allProperties (/home/orestis/.atom/packages/api-workbench/main.js:10:26388)
    at t.allProperties (/home/orestis/.atom/packages/api-workbench/main.js:3:8636)
    at e.process (/home/orestis/.atom/packages/api-workbench/main.js:20:22310)
    at e.process (/home/orestis/.atom/packages/api-workbench/main.js:20:21791)
    at t.children (/home/orestis/.atom/packages/api-workbench/main.js:3:29308)
    at t.attrs (/home/orestis/.atom/packages/api-workbench/main.js:3:30609)
    at t.attr (/home/orestis/.atom/packages/api-workbench/main.js:3:30248)
    at /home/orestis/.atom/packages/api-workbench/main.js:7:18092
    at Array.forEach (native)
    at e.createStubNode (/home/orestis/.atom/packages/api-workbench/main.js:7:18013)
    at Object.a [as createStub] (/home/orestis/.atom/packages/api-workbench/main.js:29:23590)
    at Object.v [as newNode] (/home/orestis/.atom/packages/api-workbench/main.js:29:15982)
    at Object.onClick (/home/orestis/.atom/packages/api-workbench/main.js:42:21250)
    at e.onClick (/home/orestis/.atom/packages/api-workbench/main.js:31:27335)
    at /home/orestis/.atom/packages/api-workbench/main.js:31:20606
    at /home/orestis/.atom/packages/api-workbench/main.js:27:18055
    at Array.forEach (native)
    at HTMLButtonElement._onClickListeners.length._onAltClickListeners.length.e.onclick (/home/orestis/.atom/packages/api-workbench/main.js:27:18028)
```

### Commands

```
     -2:46.8.0 api-workbench:new-project (atom-pane.pane.active)
     -2:29.1.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -2:28.8.0 core:paste (atom-text-editor.editor.mini.is-focused)
  2x -2:22.8.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -2:14 core:move-right (atom-text-editor.editor.mini.is-focused)
  3x -2:13.8.0 core:delete (atom-text-editor.editor.mini.is-focused)
 30x -1:03.0 core:backspace (atom-text-editor.editor.is-focused)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
api-workbench, v0.8.9
linter, v1.11.3

# Dev
No dev packages
```

**Comment by ddenisenko** [comment]
Thank you for reporting this, we will investigate.

**Comment by shankaramarapini** [comment]
can some one give solution for the issue:(

**Comment by ddenisenko** [comment]
Well, the fix for the issue itself will be published at the Monday release tomorrow.

For now you can simply copy-paste the code from the next RAML snippet in the tutorial (which displays what happens after you invoke the Palette action):

```RAML
#%RAML 1.0
title: Pet shop
version: 1
baseUri: /shop

/pets:
  get:
    responses:
      200:
        body:
          application/json:

  post:
    body:
      application/json:

```
Or type the same code. You don't have to use the actions from Palette for everything, typing is also a viable approach.

**Comment by shankaramarapini** [comment]
Cool . Thanks

**Comment by ddenisenko** [comment]
Fixed. Fix should be available in 0.8.10.
